### ISSUE TITLE
V1.1.20: playback broken

### ISSUE THREAD

**Issue by erkr** [music-assistant/hass-music-assistant#272]
When adding tracks to the queue, playback immediately stops, both for Spotify tracks and local media.

Error with Spotify:
```
Logger: homeassistant
Source: /usr/src/homeassistant/homeassistant/runner.py:96 
First occurred: 20:33:59 (2 occurrences) 
Last logged: 20:33:59

Error doing job: Future exception was never retrieved
Traceback (most recent call last):
  File "/usr/local/lib/python3.9/site-packages/music_assistant/helpers/process.py", line 69, in __aexit__
    await self._proc.stdin.drain()
  File "/usr/local/lib/python3.9/asyncio/streams.py", line 387, in drain
    await self._protocol._drain_helper()
  File "/usr/local/lib/python3.9/asyncio/streams.py", line 197, in _drain_helper
    await waiter
BrokenPipeError
```
Local media:
```

Logger: homeassistant
Source: /usr/src/homeassistant/homeassistant/runner.py:96 
First occurred: 20:32:02 (2 occurrences) 
Last logged: 20:32:47

Error doing job: Future exception was never retrieved
BrokenPipeError
```

**Comment by marcelveldt** [comment]
Does it make any difference which player you pick ? I can't reproduce the issue. Everything plays fine and tested on multiple players, including Chromecast.

BrokenPipeError means that the stream was disrupted

**Comment by erkr** [comment]
As just discussed via Discord: had two issues:
- One: in my case player queues got corrupted in the previous version (due extension with capitals). Solved after clearing all queues and a restart!
- two: casting to a group causes casting errors. To a single CC device worked (not tested thoroughly)

**Comment by marcelveldt** [comment]
released 1.1.21 patch release with a fix

**Comment by erkr** [comment]
First impression: fixed 🎊🥳
